packages/vertexai/src/errors.ts

```diff
@@ -34,15 +34,15 @@ export class VertexAIError extends FirebaseError {
    */
   constructor(
     readonly code: VertexAIErrorCode,
-    readonly message: string,
+    message: string,
```

Comment:
Sorry, I know I reviewed this already, but is there a reason that this isn't readonly anymore?

Reply:
This is a good question, I was going to mention it in the description. If it's readonly it can't be overwritten by super() which was causing some unexpected behavior. super() at the bottom of the constructor sets `message` to fullMessage (adds the service name and code) but it simply won't do anything if it's readonly. The next question would be, what about `code` - it's ok for code to not be overwritten with fullCode because adding the prefix just makes it harder to compare (e.g. `error.code === VertexAIErrorCode.SOME_CODE`). The formatted `fullCode`. only needs to go into the message.